composer.json

```diff
@@ -13,21 +13,19 @@
         }
     ],
     "require": {
-        "php": "^5.5|^7.0",
+        "php": "^7.2",
```

Comment:
Why not 7.1? Are you using any specific language feature?

Reply:
jms/serializer requires 7.2, and this lib fully depends on it